endpoints/put.go

```diff
@@ -87,7 +87,12 @@ func NewPutHandler(backend backends.Backend, maxNumValues int, allowKeys bool) f
 			}
 
 			logrus.Debugf("Storing value: %s", toCache)
-			resps.Responses[i].UUID = uuid.NewV4().String()
+			var uuidGenError error
+			resps.Responses[i].UUID, uuidGenError = utils.GenerateUUIDString()
```

Comment:
You could save a line by using `:=` here I believe. I think you just need one undefined variable in the group for it to be valid go. But it is also fine the way it is.

delete line 90 and 91 is:
`resps.Responses[i].UUID, uuidGenError := utils.GenerateUUIDString()`

Reply:
I made the proposed change and I'm getting:
```
endpoints/put.go|92 col 4| : expected identifier on left side of :=
```

Reply:
That is odd, perhaps the UUID being burried in a structure is confusing things.

Reply:
@hhhjort are you ok with leaving these lines as they are? Do you want me to try something else?

Reply:
Should be fine then